plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/procedure/MigrateProcedure.java

```diff
@@ -303,7 +304,7 @@ private Schema toIcebergSchema(List<Column> columns)
         List<Types.NestedField> icebergColumns = new ArrayList<>();
         for (Column column : columns) {
             int index = icebergColumns.size();
-            org.apache.iceberg.types.Type type = toIcebergType(typeManager.getType(column.getType().getTypeSignature()), nextFieldId);
+            org.apache.iceberg.types.Type type = toIcebergType(typeManager.getType(column.getType().getTypeSignature(DEFAULT_PRECISION)), nextFieldId);
```

Comment:
#17391 is going to use `MICROSECONDS`. Can we close this PR so we can avoid unnecessary conflicts on the PR?